src/relay/backend/aot_executor_codegen.cc

```diff
@@ -317,6 +327,16 @@ class AOTExecutorCodegen : public MixedModeVisitor {
     }
   }
 
+  void PushArgs(const Expr& expr, std::vector<tir::Var> sids, Array<PrimExpr>* args) {
```

Comment:
nit : any reason mixed pass-by-value, pass-by-(const)-references usage and pointers here ?